docs/self-managed/components/management-identity/configuration/connect-to-an-oidc-provider.md

```diff
@@ -17,6 +17,10 @@ Connect Management Identity to an OpenID Connect (OIDC) authentication provider
 - A full list of supported and unsupported features when using an OIDC provider is available in the [OIDC features table](#supported-and-unsupported-oidc-features).
 - To connect to a Keycloak authentication provider, see [connect to an existing Keycloak instance](/self-managed/components/management-identity/configuration/connect-to-an-existing-keycloak.md).
 
+:::info Deploying with Helm?
+If you deploy Camunda 8 Self-Managed with Helm, use the [Helm chart authentication and authorization guides](/self-managed/deployment/helm/configure/authentication-and-authorization/index.md) to configure OIDC and Management Identity.
+
+This guide focuses on application-level Management Identity configuration (for example, environment variables and IdP application settings) and applies regardless of deployment method.
```

Comment:
The part "applies regardless of deployment method" would be a bit confusing for me as a reader, because it sounds like I should follow this guide in any case. Maybe we can replace this with something like "Follow this guide if you want to configure Management Identity on the application level or if you would like to understand how configuration properties apply at the application level" (I'm sure there's a more compact way to say this :))?